src/main/java/org/kiwiproject/dropwizard/error/job/CleanupApplicationErrorsJob.java

```diff
@@ -0,0 +1,38 @@
+package org.kiwiproject.dropwizard.error.job;
+
+import lombok.extern.slf4j.Slf4j;
+import org.kiwiproject.dropwizard.error.config.CleanupConfig;
+import org.kiwiproject.dropwizard.error.dao.ApplicationErrorDao;
+
+import java.time.ZonedDateTime;
+
+/**
+ * Job that can be configured to run on a regular interval that will delete expired application errors.
+ *
+ * @see CleanupConfig for options on what will be deleted and when
+ */
+@Slf4j
+public class CleanupApplicationErrorsJob implements Runnable {
```

Comment:
Should this implement `CatchingRunnable` instead, so that if the job ever fails, it won't cause a `ScheduledExecutorService` to stop?